src/parsec_client/operations/list_keys.md

```diff
@@ -0,0 +1,37 @@
+# ListKeys
+
+Lists all keys belonging to the application. Opcode: 26 (`0x001A`)
+
+## Parameters
+
+No parameters are needed for this operation.
+
+## Results
+
+| Name   | Type                                 | Description             |
+|--------|--------------------------------------|-------------------------|
+| `keys` | Vector of [`KeyInfo`](#keyinfo-type) | List of key information |
+
+### KeyInfo type
+
+A `KeyInfo` type contains the following members:
+
+| Name          | Type                                                        | Description                        |
+|---------------|-------------------------------------------------------------|------------------------------------|
+| `provider_id` | Unsigned integer                                            | ID of the Provider holding the key |
+| `name`        | String                                                      | Name of the key                    |
+| `attributes`  | [`KeyAttributes`](psa_key_attributes.md#keyattributes-type) | Attributes of the key              |
+
+## Specific response status codes
+
+No specific response status codes returned.
+
+## Description
+
+This operation lists all the keys that an application created in all providers.
```

Comment:
Does this want to be changed re @ionut-arm's suggestion?

Reply:
We decided internally that since `ListKeys` is a core operation, it should only target the Core provider 😄